ompi/datatype/ompi_datatype_create_darray.c

```diff
@@ -216,9 +216,14 @@ int32_t ompi_datatype_create_darray(int size,
     }
 
     /* Build up array */
+    displs[0] = st_offsets[start_loop];
```

Comment:
this was not supposed to get pushed. It is from another patch, but it is wrong and incomplete. I will remove.